### PR TITLE
Disclaimer

### PR DESCRIPTION
**Pull request by lyndseysc**
adding an anonymous message on the survey description

adding the discussed disclaimer onto the survey and changing the dummy data and test to reflect it

### CODE REVIEW

File: client/src/pages/TakeSurvey/SurveyDescription/SurveyDescription.jsx [hunk 1]
```diff
@@ -26,6 +44,8 @@ const SurveyDescription = () => {
       <Typography data-testid='survey-disclaimer' variant='h5'>
         {survey.disclaimer}
       </Typography>
+      {survey.anonymous === true && <AnonymousDisclaimer />}
```
Comment (Joaoviana) [inline client/src/pages/TakeSurvey/SurveyDescription/SurveyDescription.jsx]: ternary would be ok here

File: client/src/pages/TakeSurvey/TakeSurvey.test.jsx [hunk 2]
```diff
@@ -12,7 +12,8 @@ const dummySurvey = {
   description:
     'A survey description that is about 1 sentence long yeah blah blah',
   status: 'created',
-  disclaimer: 'some disclaimer',
+  disclaimer:
+    'In addition to your opinion, we will also collect your name and email address for a follow up if required. We respect your trust and protect your privacy by never sharing this data with third parties as well as ensuring this data will be stored until the research period is over. By filling out this survey, you agree that we we will process your data in line with the Beamery Privacy Policy. If you would like to chat about this policy, or if you subsequently change your mind and want us to delete your personal data, please email steven.bianchi@beamery.com.',
```
Comment (TKostrzewski) [inline client/src/pages/TakeSurvey/TakeSurvey.test.jsx]: There's a double "we" in the middle

Comment (TKostrzewski) [reply client/src/pages/TakeSurvey/TakeSurvey.test.jsx]: Could we change "as well as ensuring this data will be stored until the research period is over." to "as well as ensuring this data will not be stored after the research period has finished."

File: client/src/pages/TakeSurvey/SurveyDescription/SurveyDescription.jsx (hunk 1 above)
Comment (thomas-galligan) [reply client/src/pages/TakeSurvey/SurveyDescription/SurveyDescription.jsx]: how about `survey.anonymous && <AnonymousDisclaimer/>`?

File: client/src/pages/TakeSurvey/TakeSurvey.test.jsx (hunk 2 above)
Comment (TKostrzewski) [reply client/src/pages/TakeSurvey/TakeSurvey.test.jsx]: Is it possible to link the Privacy policy to the actual policy? Then the recipients need only speak to Steve if they don't want their data stored?

Comment (lyndseysc) [reply client/src/pages/TakeSurvey/TakeSurvey.test.jsx]: this is what was sent to us by Milo and Andrew, caught the double 'we' !

Comment (lyndseysc) [reply client/src/pages/TakeSurvey/TakeSurvey.test.jsx]: gonna add the link

**Pull request merged**